### DISCUSSION TITLE
Meetings notes and actions

### DISCUSSION THREAD

**Discussion by phildexter** [wagtail/guide#2] [General]
This is where we will store our notes from our meetings and the actions we each take away to complete.

**Comment by phildexter** [comment]
## Updates
@allcaps showed us his demo of simple-translations using en-v216, en-latest, en-v30. We discussed changing the url path to include a - or a _ so, en-v2_16. Coen was unsure if that was allowed in the path so took an action to check.

## Actions
- [x] @allcaps to check if simple-translations accepts unscores in the locale path.
- [x] @Hitansh-Shah to create a wagtail project and try out simple translations like Coen's demo
- [x] @vossisboss to research the existing documentation and the [documentation from Divio](https://documentation.divio.com/) and document which field types we might want on our documentation page.
- [ ] @phildexter to create a draft project plan

**Comment by phildexter** [comment]
Question for next meeting - what do we want to do with the old editors guide? Automatic redirects? Leave them there in the developers docs, or remove them?

**Reply by phildexter** [reply]
Another - how do we want search to work? Dropdown results or results page?

**Reply by phildexter** [reply]
Another - what are our acceptance criteria for a ticket to be done? Do we expect unit tests written per ticket? Will each ticket go through code review?

**Reply by Hitansh-Shah** [reply]
> Another - how do we want search to work? Dropdown results or results page?

I personally feel that the dropdown results are more convenient and quick. Though I am not very sure of the technical differences in implementing them.

**Reply by phildexter** [reply]
> I personally feel that the dropdown results are more convenient and quick. Though I am not very sure of the technical differences in implementing them.

Yeah I think it would be worth us looking at how some of the best documentation sites work here.

**Comment by phildexter** [comment]
Actions: 
- [x] @vossisboss to research the existing documentation and the [documentation from Divio](https://documentation.divio.com/) and document which field types we might want on our documentation page.
- [x] @phildexter and @allcaps to try out having a en-latest locale, copying the page tree and setting it up with a new locale (en-3-0)
- [x] @vossisboss and @phildexter to spec out the content model for the new page types (Wagtail Builder?!)
- [x] @phildexter do some research into how search works on other excellent documentation sites

Decisions:

> Question for next meeting - what do we want to do with the old editors guide? Automatic redirects? Leave them there in the developers docs, or remove them?

Leave them where they are, no redirects. We don't support any old versions so leave them there.

> Another - how do we want search to work? Dropdown results or results page?

You can link a search results page, but you can't with a dropdown. Basic Wagtail would be best to have a search results page, or like wagtail developer docs. More research needed.

> Another - what are our acceptance criteria for a ticket to be done? Do we expect unit tests written per ticket? Will each ticket go through code review?

A PR for each ticket, and code review and unit tests. 

We then had a long discussion about the best approach to locales and trees and versions and agreed to do some more testing.

**Reply by phildexter** [reply]
I've done some [research on search here](https://github.com/wagtail/guide/wiki/Search-research...), that we can discuss at some point.

**Comment by phildexter** [comment]
Actions:

- [ ] @allcaps to speak to the core team about who should host the final editors guide and where, and what tooling we should use
- [x] @phildexter to get updated designs for the FE based on the feedback
- [ ] @allcaps create an MR for making locales editable in the admin interface
- [x] @allcaps look into setting up Gitpod for development QA
- [x] @Hitansh-Shah respond to PR comments, setup Black and Github actions and review CookieCutter

**Comment by phildexter** [comment]
Actions:
- [x] @allcaps to speak to the core team about who should host the final editors guide and where, and what tooling we should use
- [x] @phildexter to get updated designs for the FE based on the feedback

**Comment by thibaudcolas** [comment]
Notes from today:

- Ways of working
- Now going for hosting w/ Torchbox
- wagtail.org similar situation – hosted by Torchbox, maintained by core team
- Separate Heroku account maybe to facilitate access control?
- Project backlog tracked in GitHub Projects
- Meagen chatting w/ Abigail about launch – launch checklist
- Gitpod working quite well, setup by Coen w/ demo content
- Homepage almost there. Currently in progress on search
- Next - component pages
- Coen draft PR for table of contents, unsure of approach
- Workshop to confirm design / content model
- Coen has done some of the site setup, went through it w/ Hitansh
- Hitansh feeling familiar but not from scratch
- Next steps? 
- Feature development – where are we at?
- Translation, versioning, 
- Locale setup only done by Coen (w/ review from Hitansh), interested in doing more of that
- Tests – added basic response code checks, template check, rendering of blocks
- Coen then adding factories, refactoring
- Planning – ok to stay away from this
- Designing & front-end – doing this a bit on side projects
- Unsure how much of a need we have for locales in the short term
- Same for versioning – feels valuable but not a must for now
- Backlog review / planning
- hreflang/canonical in progress, feedback ready for tackle

We added all actions to the [project backlog](https://github.com/orgs/wagtail/projects/4/views/1), alongside other tasks.
